src/main/java/mate/academy/spring/mapper/impl/request/OrderRequestMapper.java

```diff
@@ -0,0 +1,31 @@
+package mate.academy.spring.mapper.impl.request;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import mate.academy.spring.mapper.DtoRequestMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.OrderRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderRequestMapper implements DtoRequestMapper<OrderRequestDto, Order> {
```

Comment:
this mapper redundant
see https://mate-academy.github.io/jv-program-common-mistakes/java-spring/rest-practice/jv-spring-rest-practice_checklist